benefits/core/models/transit.py

```diff
@@ -78,6 +78,72 @@ def __str__(self):
         return f"({environment_label}) {agency_slug}"
 
 
+class SwitchioConfig(models.Model):
+    """Configuration for connecting to Switchio, an entity that applies transit agency fare rules to rider transactions."""
+
+    id = models.AutoField(primary_key=True)
+    environment = models.TextField(
+        choices=Environment,
+        help_text="A label to indicate which environment this configuration is for.",
+    )
+    api_key = models.TextField(help_text="The API key used to access the Switchio API.", default="", blank=True)
+    api_secret_name = SecretNameField(
+        help_text="The name of the secret containing the api_secret value used to access the Switchio API.",  # noqa: E501
+        default="",
+        blank=True,
+    )
+    client_certificate = models.ForeignKey(
+        PemData,
+        related_name="+",
+        on_delete=models.PROTECT,
+        help_text="The client certificate for accessing the Switchio API.",
+        null=True,
+        blank=True,
+        default=None,
+    )
+    ca_certificate = models.ForeignKey(
+        PemData,
```

Comment:
This is one of those things we probably can't know for sure until we can hit the API, but...

In the old days with the legacy Littlepay API, we actually needed 3 `PemData` fields. See my comment here for more: https://github.com/cal-itp/benefits/issues/2848#issuecomment-2810515567

Reply:
Added `private_key` field in https://github.com/cal-itp/benefits/pull/2857/commits/acf0884bae1437f12ac62802ffb3e80bf026c674